.github/workflows/manage-runner-post/action.yaml

```diff
@@ -1,64 +0,0 @@
-name: Setup runner
-description: Reusable action for tearing down the github runner
-
-inputs:
-  GITHUB_TOKEN:
-    description: 'Automatically assigned GitHub Token'
-    required: true
-
-runs:
-  using: composite
-  steps:
-    ########################################
-    # Stop bazel-remote cache
-    ########################################
-    - name: "💾 Stop bazel-remote s3 cache"
-      run: ./bin/bazel-cache-s3-stop.sh
```

Comment:
If we decide to delete the workflows in the end, we don't need these scripts either.